src/modules/holder/pages/ConfirmSignIn/useHolderConfirmSignIn.tsx

```diff
@@ -57,5 +60,11 @@ export const useHolderConfirmSignIn = () => {
     }
```

Comment:
Wouldn't it be more convenient to wrap this into try/catch instead of making a separate useEffect for setIsLoading():

```ts
try {
  await mutateAsync({
    token: storage.getItem('signUpToken') || '',
    confirmationCode: computedCode,
  })
} finally {
  setIsLoading(false)
}
```

Reply:
omg, it should be `catch`. autocomplete failed me  😅

Reply:
I'm not sure that `useMutation` will throw an error because it is handle errors under the hood

Reply:
fixed